include/tbl/params.h

```diff
@@ -732,6 +732,18 @@ PARAM_SIMPLE(
 	/* flags */	EXPERIMENTAL
 )
 
+PARAM_SIMPLE(
+	/* name */	panic_buffer,
+	/* type */	bytes_u,
+	/* min */	"4k",
+	/* max */	"1g",
+	/* def */	"64k",
+	/* units */	"bytes",
+	/* descr */
+	"Size of the panic message buffer.",
```

Comment:
Maybe add a note here that enough space where shared memory is stored is required.
I am not sure about the wording, but a small note here would help.